content/remote-development/release-notes.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Release Notes v0.1.0
+menu:
+  main:
+    parent: 'remote-development'
+    weight: 50
+
+weight: 50
```

Comment:
This could do with being either the first or last item, it looks a bit odd being ordered in the middle of the reference material.